test/extended/networking/services.go

```diff
@@ -131,13 +138,19 @@ var _ = Describe("[sig-network] services", func() {
 			By("check load balance service creation fails")
 			for {
 				err := createWebserverLBService(k8sClient, namespace, serviceName, "", []string{"192.168.132.10"}, nil)
+				e2e.Logf("error occurred while creating %s/%s service: %v: reason: %v", namespace, serviceName, err, kapierrs.ReasonForError(err))
+				if err != nil && strings.Contains(err.Error(), "connection reset by peer") {
+					time.Sleep(retryInterval)
+					k8sClient = oc.KubeClient()
+					serviceClient = k8sClient.CoreV1().Services(namespace)
+					continue
```

Comment:
Can you clarify how this will help perhaps with a comment? I would normally expect us to be retrying the createWebserverLBService on error, not reassigning the client vars after a sleep.

Reply:
@dgoodwin this is for handling `connection reset by peer ` [error](https://prow.ci.openshift.org/spyglass/lens/junit/iframe?req=%7B%22artifacts%22%3A%5B%22artifacts%2Fe2e-aws-serial%2Fgather-extra%2Fartifacts%2Fjunit%2Fjunit_install_status.xml%22%2C%22artifacts%2Fe2e-aws-serial%2Fgather-extra%2Fartifacts%2Fjunit%2Fjunit_symptoms.xml%22%2C%22artifacts%2Fe2e-aws-serial%2Fgather-must-gather%2Fartifacts%2Fjunit_install.xml%22%2C%22artifacts%2Fe2e-aws-serial%2Fopenshift-e2e-test%2Fartifacts%2Fjunit%2Fjunit_e2e_20220427-183141.xml%22%2C%22artifacts%2Fe2e-aws-serial%2Fopenshift-e2e-test%2Fartifacts%2Fjunit_nodes.xml%22%2C%22artifacts%2Fjunit_operator.xml%22%5D%2C%22index%22%3A2%2C%22src%22%3A%22gs%2Forigin-ci-test%2Flogs%2Fperiodic-ci-openshift-release-master-ci-4.11-e2e-aws-serial%2F1519354700044963840%22%7D&topURL=https%3A//prow.ci.openshift.org/view/gcs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.11-e2e-aws-serial/1519354700044963840&lensIndex=2#) from kube api server and it happens when client trying to create service while kube api server rolling out external ip config. i think something bad happens at api server for a while and closes the client connection, so shouldn't we reestablish  client connection (by reassigning vars here) and try it again ?

Reply:
Sorry one more request, can you Logf right before the sleep explaining why we're sleeping up to 30 mins here, might throw someone off watching their CI logs thinking something's jammed.

Reply:
One more request here @pperiyasamy plz

Reply:
oh! yes @dgoodwin , done.